dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -216,28 +218,37 @@ def _add_helper(self, other1, other2):
         if "sum" in self.__calculations.keys():
             self.sum = other1.sum + other2.sum
         if "skewness" in self.__calculations.keys():
-            self._biased_skewness = self._merge_biased_skewness(other1.match_count,
```

Comment:
need to add the merge of `self.bias_correction` if two different give a warning and set the bias_correction False